doc/corelib/index-list.html.template

```diff
@@ -0,0 +1,188 @@
+
+<h1>The Coq Prelude</h1>
+
+<p>Here is a short description of the Coq Prelude, which is
+distributed with the system.
+It provides a set of modules directly available
+through the <tt>Require Import</tt> command.</p>
+
+<p>The prelude is composed of the following subdirectories:</p>
+
+<dl>
+  <dt> <b>Init</b>:
+    The core library (automatically loaded when starting Coq)
```

Comment:
I wonder if we shouldn't switch the titles here: The Core library at the beginning, and the Prelude in the description of `Init`.